f/zeus/fn_zeusAddAddons.sqf

```diff
@@ -41,25 +41,15 @@ if (isNull _curator || typeOf _curator != "ModuleCurator_F") exitWith {
 
 // Decide which addons to add based on passed mode
 _addons = [""];
-_curator setVariable ["Addons",0,true];
 
 switch (typeName _mode) do {
 	case "ARRAY": {_addons = _mode};
 	case "STRING": {_addons = [_mode]};
 	case "BOOL": {
 		if (_mode) then {
 			_curator setVariable ["Addons",3,true];
```

Comment:
Might be worth an additional comment to mention this relies on / sets the ModuleCurator_F module initialisation to the "all addons + unofficlal" option